kuttl/tests/upgrade/clustered-deployment-c3/00-deploy-splunk-operator-1-0-5.yaml

```diff
@@ -0,0 +1,7 @@
+#create new cluster manager custom resource 
```

Comment:
Taking a step back, the 1.0.x to 1.1 Upgrade is a special one for us due to the major SDK upgrade. We don't expect this special case upgrade to occur once we move on to future releases. 

I guess I am questioning the requirement for these extensive tests being added to the code base (50+ files) unless we intend to use some of these for the future upgrades. Can you please clarify?

Reply:
yes the idea is to use  this to use for future upgrades. if its not needed I can remove them.